### ISSUE TITLE
Wifi Hotspot doesn't work 

### ISSUE THREAD

**Issue by janengelmohr** [visi0nary/android_device_alps_k05ts_a#36]

**Comment by walter79** [comment]
```
<string-array name="config_tether_usb_regexs">
    <item>rndis\\d</item>
</string-array>

<string-array name="config_tether_wifi_regexs">
    <item>ap\\d</item>
</string-array>

<string-array translatable="false" name="config_tether_bluetooth_regexs">
    <item>"bt-pan"</item>
    <item>"bt-dun"</item>
</string-array>
```

**Comment by janengelmohr** [comment]
Yep, saw that overlay in hermes device tree. Will build it now :)

**Comment by janengelmohr** [comment]
Fixed with f6b36a75e7fb805d9dee65af5469c13b2bb97e3f
